internal/services/powerbi/powerbi_embedded_resource_test.go

```diff
@@ -180,7 +180,7 @@ resource "azurerm_powerbi_embedded" "test" {
   name                = "acctestpowerbi%[2]d"
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
-  sku_name            = "A1"
+  sku_name            = "A7"
```

Comment:
this doesn't need to be included in a test, as long as you have confirmed the new sku values are valid that is fine.

Reply:
Sure. I confirmed that new value is valid since it works fine on my local.